pkg/storage/governance_request_feedback.go

```diff
@@ -0,0 +1,129 @@
+package storage
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/google/uuid"
+	"go.uber.org/zap"
+	"golang.org/x/exp/slices"
+
+	"github.com/cmsgov/easi-app/pkg/appcontext"
+	"github.com/cmsgov/easi-app/pkg/apperrors"
+	"github.com/cmsgov/easi-app/pkg/models"
+)
+
+// CreateGovernanceRequestFeedback creates a governance request feedback record in the database
+func (s *Store) CreateGovernanceRequestFeedback(ctx context.Context, requestFeedback *models.GovernanceRequestFeedback) (*models.GovernanceRequestFeedback, error) {
+	// validate first
+	isFeedbackValid := validateSourceActionTargetFormCombination(*requestFeedback)
+	if !isFeedbackValid {
+		errorMessage := fmt.Sprintf(
+			"Invalid combination of source action and target feedback; %v is an invalid target form for a %v action",
+			requestFeedback.TargetForm,
+			requestFeedback.SourceAction,
+		)
+		appcontext.ZLogger(ctx).Error(errorMessage)
+		return nil, errors.New(errorMessage)
+	}
+
+	if requestFeedback.ID == uuid.Nil {
+		requestFeedback.ID = uuid.New()
+	}
+
+	const governanceRequestFeedbackCreateSQL = `
+		INSERT INTO governance_request_feedback(
+			id,
+			intake_id,
+			feedback,
+			source_action,
+			target_form,
+			created_by
+		) VALUES (
+			:id,
+			:intake_id,
+			:feedback,
+			:source_action,
+			:target_form,
+			:created_by
+		) RETURNING *;
+	`
+
+	stmt, err := s.db.PrepareNamed(governanceRequestFeedbackCreateSQL)
+	if err != nil {
+		appcontext.ZLogger(ctx).Error(
+			fmt.Sprintf("Failed to create governance request feedback with error %s", err),
+			zap.Error(err),
+			zap.String("user", requestFeedback.CreatedBy),
+		)
+		return nil, err
+	}
+
+	retFeedback := models.GovernanceRequestFeedback{}
+
+	err = stmt.Get(&retFeedback, requestFeedback)
+	if err != nil {
+		appcontext.ZLogger(ctx).Error(
+			fmt.Sprintf("Failed to create governance request feedback with error %s", err),
+			zap.Error(err),
+			zap.String("user", requestFeedback.CreatedBy),
+		)
+		return nil, err
+	}
+
+	return &retFeedback, nil
+}
+
+// GetGovernanceRequestFeedbacksByIntakeID returns all governance request feedback items for a given system intake
+func (s *Store) GetGovernanceRequestFeedbacksByIntakeID(ctx context.Context, intakeID uuid.UUID) ([]*models.GovernanceRequestFeedback, error) {
+	feedbacks := []*models.GovernanceRequestFeedback{}
+
+	stmt, err := s.db.PrepareNamed(`SELECT * FROM governance_request_feedback WHERE intake_id = :intake_id`)
```

Comment:
What do you think about explicitly listing out the columns for the `SELECT` instead of `SELLECT *`?  We've have a couple issues where `SELECT *` creates a transient error in the app during deployment (because the database is updated before the app code is updated). This paradigm certainly has precedence in the `EASi` code base if you prefer it.

Reply:
I do want to think about this in general; I still need to read over what you posted in Slack about this discussion. I think I'm going to stay with `SELECT *` on this particular PR to just keep things moving, but I'm definitely open to revisiting our approach.

Reply:
Thanks Dylan! Yeah that sounds perfect to me.